tools/e2e-tests/apps/assets/client/main.html

```
@@ -0,0 +1,22 @@
+<head>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)